ai-eval-service/evaluator.py

```
@@ -121,14 +100,11 @@ async def evaluate_response(
-        category_mapping = {
-            "gender": "gender",
-            "race": "race",
-            "ethnicity": "race",
-            "religion": "religion",
-            "age": "age",
-        }
-        langfair_category = category_mapping.get(category.lower(), "gender")
+        category_lower = category.lower()
+        if category_lower not in self.CATEGORY_MAPPING:
+            valid_categories = ', '.join(sorted(self.CATEGORY_MAPPING.keys()))
+            raise ValueError(f"Unrecognized category '{category}'. Valid categories: {valid_categories}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)